kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -3620,4 +3620,20 @@ object KyuubiConf {
       .version("1.8.1")
       .booleanConf
       .createWithDefault(false)
+
+  private val HIVE_SERVER2_THRIFT_RESULTSET_DEFAULT_FETCH_SIZE: ConfigEntry[Int] =
+    buildConf("hive.server2.thrift.resultset.default.fetch.size")
+      .internal
+      .serverOnly
+      .intConf
+      .createWithDefault(1000)
```

Comment:
let's add doc and version too. as this is a internal conf, we can write some tech details in doc field

Reply:
version